src/routes/search/components/NothingFoundFeedbackBox.js

```diff
@@ -57,6 +70,14 @@ class NothingFoundFeedbackBox extends React.Component<PropsType, StateType> {
                 buttonStyle={{backgroundColor: theme.colors.themeColor}}
                 onPress={this.onSubmit} title={t('feedback:send')} />
       </>
+    } else if (sendingStatus === 'sending') {
+      return <ActivityIndicator size='large' color='#0000ff' />
+    } else { // sendingStatus === 'successful'
```

Comment:
Can possibly be done using a switch case which will show a flow error if a case is missing.

Reply:
Nope it can't